deps/libuv.version

```diff
@@ -1,2 +1,2 @@
-LIBUV_BRANCH=julia-uv1.9.0
-LIBUV_SHA1=ecbd6eddfac4940ab8db57c73166a7378563ebd3
+LIBUV_BRANCH=jn/pid-zero
+LIBUV_SHA1=cb6d0f875a5b8ca30cba45c0c1ef7442c87c1e68
```

Comment:
that's **exactly** what the WIP tag is intended to convey

> The convention is to prefix the pull request title with "WIP:" for Work In Progress, or "RFC:" for Request for Comments when work is completed and ready for merging. This will prevent accidental merging of work that is in progress.